src/components/Progress.js

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { observer } from "mobx-react"
+
+import { Button, Link, List, InlineBlock, Heading, Provider, Divider } from "reakit";
+import theme from "reakit-theme-default";
+//import { blue } from "../colors"
+
+import CircularProgressbar from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+import styled from "styled-components"
+import Layout from "../layouts/Text"
+
+
+// Need to make changes here
```

Comment:
Can you add detail?

Reply:
I added a TODO and referenced #42 for future work.